man/pdfpcrc.in

```diff
@@ -9,9 +9,6 @@
 .SH NAME
 pdfpcrc \- pdfpc configuration file
 
-.SH SYNOPSIS
```

Comment:
This should rather go to a FILES sections?

Reply:
You are right, yes.

Reply:
Please check now.